tools/RAiDER/processWM.py

```diff
@@ -83,11 +83,6 @@ def prepareWeatherModel(
     f = weather_model.load()
 
     if f is not None:
-        logger.warning(
-            'The processed weather model file already exists,'
-            ' so I will use that.'
-        )
-
```

Comment:
Again, why are you removing this? Should it be included in this PR.

Reply:
it's misleading because if the processed weather model already exists, the warning is raised on line 58.